app/messages/page.jsx

```diff
@@ -1,5 +1,58 @@
-import styles from '@/styles/Home.module.css';
+'use client';
+
+// This will use useContext when we have more set up in our app
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
```suggestion
import React, { useEffect, useState } from 'react';
```
Import useGlobalContext when we're using it